projects/DensePose/doc/TOOL_APPLY_NET.md

```diff
@@ -35,6 +35,67 @@ python apply_net.py dump configs/densepose_rcnn_R_50_FPN_s1x.yaml DensePose_ResN
 ```
 
 
+
+If you want to load the pickle file of DensePose model results: 
+```
+f = open('/your_result_path/result.pkl', 'rb')
+data = pickle.load(f)
+```
+
+
+The pickle loading code should be run in the `/your_detectron2_path/detectron2_repo/projects/DensePose/` directory, 
+or `ModuleNotFoundError: No module named 'densepose'` will occur. 
+
+If you have to run the code in other directory, simply add this before pickle loading:
+```
+import sys
+sys.path.append("/your_detectron2_path/detectron2_repo/projects/DensePose/")
+```
```

Comment:
Simplify to the following:

If you want to load the pickle file generated by the above command: 
```
# make sure DensePose is in your PYTHONPATH, or use the following line to add it:
sys.path.append("/your_detectron2_path/detectron2_repo/projects/DensePose/")

f = open('/your_result_path/result.pkl', 'rb')
data = pickle.load(f)
```